### ISSUE TITLE
Babel compilation using this plugin

### ISSUE THREAD

**Issue by pavel06081991** [airbnb/babel-plugin-dynamic-import-node#13]
Could you please clarify one thing. When I use babel-node with this plugin, all is okay. But if I use this plugin during babel compilation like this:

`"build": "babel lib -d dist --presets es2015,stage-2"`

will babel-plugin-dynamic-import-node work after compilation? I mean if for example I use dynamic import like this:

myLib.js:
`import express from 'express';`

index.js:
```
const dynamicModuleName = './my';
import(dynamicModuleName + 'Lib.js');
```
and compilation command:

`"build": "babel index.js -d dist --presets es2015,stage-2"`

Will babel-plugin-dynamic-import-node plugin compile myLib.js file during compilation or not?

**Comment by ljharb** [comment]
If you look at the output of your build script, does it contain "import()"?

**Comment by pavel06081991** [comment]
@ljharb, I've tested all I wanted and the result as I expected:
using babel-plugin-dynamic-import-node with babel-node(running babel-node from cli) all works great, dynamic import loads modules.

but using babel-plugin-dynamic-import-node with babel(source files compilation from cli using babel), it does not work and I think it is obvious because you can not know which file will be passed to import(moduleName), so you can not compile this file.

**Comment by ljharb** [comment]
Obviously you have to ensure that any file you dynamically import is properly compiled and available; thats not something babel can solve for you.

**Comment by anmonteiro** [comment]
This might have been fixed by #16
